etc/profile-m-z/zoom.profile

```diff
@@ -22,9 +22,11 @@ noblacklist ${HOME}/.zoom
 nowhitelist ${DOWNLOADS}
 
 mkdir ${HOME}/.cache/zoom
+mkfile ${HOME}/.config/zoom.conf
 mkfile ${HOME}/.config/zoomus.conf
 mkdir ${HOME}/.zoom
 whitelist ${HOME}/.cache/zoom
+whitelist ${HOME}/.config/zoom.conf
```

Comment:
No blacklist + noblacklist for it?

Reply:
Tried to keep it in sync with the rest of the profile without having the feeling of completely rewriting it (and maybe messing up in the process).

Reply:
To clarify:

* Add `blacklist ${HOME}/.config/zoom.conf` to disable-programs.inc, to prevent
  other programs from accessing it
* Add `noblacklist ${HOME}/.config/zoom.conf` to zoom.profile, to allow zoom to
  access it

See also [profile.template](https://github.com/netblue30/firejail/blob/master/etc/templates/profile.template).